### ISSUE TITLE
specifying --fast-local causes bowtie2 to always fail

### ISSUE THREAD

**Issue by nivekkagicom** [BenLangmead/bowtie2#182]
All other presets work correctly. Simply specifying --fast-local fails every time.
```
Error parsing alignment policy setting 'SEED'; RHS must have 1 token, had 2.  Policy: 'SEED=0;SEEDLEN=20;DPS=15;ROUNDS=2;IVAL=S,1,0.75;SEED=0,22;DPS=10;ROUNDS=2;IVAL=S,1,1.75'
Error: Encountered internal Bowtie 2 exception (#1)
Command: bowtie2-align-s --wrapper basic-0 --threads 8 --fast-local --gbar 4 --dovetail -x REF_2 --passthrough -U SRR015575_subset.fastq 
(ERR): bowtie2-align exited with value 1
Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid
[mpileup] 1 samples in 1 input files
<mpileup> Set max per-file depth to 8000
Use of uninitialized value $l in numeric lt (<) at /Applications/MacVector 16.5 dev/MacVector.app/Contents/Resources/bin/vcfutils.pl line 566, <> line 31.
Use of uninitialized value $l in numeric lt (<) at /Applications/MacVector 16.5 dev/MacVector.app/Contents/Resources/bin/vcfutils.pl line 566, <> line 31.

```

**Comment by rgiannico** [comment]
Exact same problem here. While waiting for the fix I'm using --local instead of --fast-local .

**Comment by ch4rr0** [comment]
I committed a fix for the change and will be working on a build which includes it

**Comment by rgiannico** [comment]
@ch4rr0  cool! thank you 😄

**Comment by ch4rr0** [comment]
https://github.com/BenLangmead/bowtie2/releases/tag/v2.3.4.2-alpha -- Here's an alpha version of the build containing the changes

**Comment by ch4rr0** [comment]
A fix for this has been released as part of our most recent version, v2.3.4.2.
